sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapi/AADResourceServerConfiguration.java

```diff
@@ -84,7 +90,9 @@ public JwtDecoder jwtDecoder() {
      */
     @Configuration
     @EnableWebSecurity
+    @EnableGlobalMethodSecurity(prePostEnabled = true)
     @ConditionalOnMissingBean(WebSecurityConfigurerAdapter.class)
+    @ConditionalOnExpression("${azure.activedirectory.enable-web-app-resource-server:false} == false")
```

Comment:
isn't using `org.springframework.boot.autoconfigure.condition.ConditionalOnProperty` simpler?

Reply:
OK, let me update it.

Reply:
I have simplified the part annotation because the `@ConditionalOnExpression` can have a default value, this is used to keep only one condition matching for each custom condition, it's not necessary, but this way the logic is clearer.